dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/bin/start.sh

```diff
@@ -19,11 +19,11 @@
 BIN_DIR=$(dirname $0)
 DOLPHINSCHEDULER_HOME=${DOLPHINSCHEDULER_HOME:-$(cd $BIN_DIR/..; pwd)}
 
-source "$DOLPHINSCHEDULER_HOME/conf/dolphinscheduler_env.sh"
 
 JAVA_OPTS=${JAVA_OPTS:-"-server -Duser.timezone=${SPRING_JACKSON_TIME_ZONE} -Xms1g -Xmx1g -Xmn512m -XX:+PrintGCDetails -Xloggc:gc.log -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=dump.hprof"}
 
 if [[ "$DOCKER" == "true" ]]; then
+  source "$DOLPHINSCHEDULER_HOME/conf/dolphinscheduler_env.sh"
```

Comment:
This may cause start error in cluster mode.

Reply:
This line maybe in ```if [[ "$DOCKER" != "true" ]]; then```